src/application.py

```diff
@@ -3,13 +3,13 @@
 from fastapi.middleware.cors import CORSMiddleware
 
 from src.api.router import api_router
-from src.bot.bot import start_bot
 from src.core.logging.middleware import LoggingMiddleware
 from src.core.logging.setup import setup_logging
 from src.settings import settings
+from src.run_bot import startup_bot, shutdown_bot
 
 
-def create_app() -> FastAPI:
+def create_app(run_bot: bool) -> FastAPI:

```

Comment:
Тут поставь дефолтое значение, чтобы можно было запускать как раньше.
```suggestion
def create_app(run_bot: bool = True) -> FastAPI:

```